--- tsc.js	2014-11-11 19:29:11.000000000 +0100
+++ tsc-patched.js	2014-11-11 19:29:44.000000000 +0100
@@ -47,7 +47,7 @@
   return exec('-v', options, function (err, stdout, stderr) {
     if (err) return callback(err, null);
 
-    var version = stdout && stdout.match(/^Version (\d+\.\d+\.\d+\.\d+)/);
+    var version = stdout && stdout.match(/Version (\d+\.\d+\.\d+\.\d+)/);
     version = version && version[1];
 
     callback(null, version);
